x-pack/plugins/stack_connectors/public/connector_types/jira/jira_params.tsx

```diff
@@ -381,6 +383,51 @@ const JiraParamsFields: React.FunctionComponent<ActionParamsProps<JiraActionPara
             }
           )}
         />
+        <EuiFormRow
+          fullWidth
+          error={errors['subActionParams.incident.otherFields']}
+          isInvalid={
+            errors['subActionParams.incident.otherFields'] !== undefined &&
+            errors['subActionParams.incident.otherFields'].length > 0
+          }
+        >
+          <JsonEditorWithMessageVariables
+            messageVariables={messageVariables}
+            paramsProperty={'otherFields'}
+            inputTargetValue={actionParams.subActionParams?.incident.otherFields || undefined}
```

Comment:
Should we do `actionParams.subActionParams?.incident?.otherFields`

Reply:
if `subActionParams` is `undefined`, it's initialized to an object containing `incident` which means there is no need for `indicent?`. Also if `subActionParams` is `undefined`, `subActionParams?` will return `undefined` so I would say we can get rid off `|| undefined`. Therefore I'll just remove the `|| undefined` part

Reply:
Does the initialization happen on the first render before the code reaches this line? Or the initialization will be available on the second render?

Reply:
I looked at the wrong code, I'll use `actionParams.subActionParams?.incident?.otherFields` as you suggested. Thanks!